scripts/reset-password.sh

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env bash
+
+#
+# Dependencies
+#
+# - curl
+# - expect
+# - jq
+# - bw (BitWarden CLI)
+# - scalingo (Scalingo CLI)
+# - xkcdpass (password generator)
+#
+# To run this script, we expect the following environment variables to be defined:
+#
+# Vaultwarden
+#
+# export BW_CLIENTID
+# export BW_CLIENTSECRET
+# export BW_PASSWORD
+#
+# Brevo
+#
+# export BREVO_API_KEY
+#
+
+set -eo pipefail
```

Comment:
```suggestion
set -euo pipefail
```

Reply:
I removed it as I am checking command usage. Using `-u` option prevent raising an explicit error message.